perfmetrics/scripts/testing_on_gke/examples/run-gke-tests.sh

```diff
@@ -306,12 +306,7 @@ function installDependencies() {
     apt-cache policy docker-ce
     sudo apt install docker-ce -y
   fi
-  # Install mash, as it is needed for fetching cpu/memory values for test runs
-  # in cloudtop. Even if mash install fails, don't panic, go ahead and install
-  # google-cloud-monitoring as an alternative.
-  which mash || sudo apt-get install -y monarch-tools || true
-  # Ensure that gcloud monitoring tools are installed. This is alternative to
-  # mash on gce vm.
+  # Ensure that gcloud monitoring tools are installed.
   # pip install --upgrade google-cloud-storage
```

Comment:
Why do we have these commented out lines of code? Should we remove them? It can be done in a separate PR.

Reply:
I think I had commented out these commands after having run these on my cloudtop/vm. I've re-enabled them to avoid any user running into an issue without them.